app/components/avatar-list.tsx

```diff
@@ -1,10 +1,23 @@
 import { take, slice } from "lodash"
 
 export type AvatarListProps = {
-  list?: { name: string; initials: string; pictureUrl?: string }[]
+  list?: AvatarType[]
+  handleOnClick: () => void
 } & JSX.IntrinsicElements["div"]
 
-function getColor(string: string) {
+export type AvatarProps = {
+  avatar: AvatarType
+  displayName?: Boolean
```

Comment:
I don't think this is being used, at least it's not used in the `Avatar` component

Reply:
That's right, I  forgot to remove it from there, thanks!